### PR TITLE
Verify license headers

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

To verify that all our source and build files must contain the correct license headers

## What are the changes implemented in this PR?

Updated dependencies to a stricter version, and added license specifications and checkstyle targets as required

### CODE REVIEW

File: dependencies/graknlabs/repositories.bzl
Comment (alexjpwalker) [inline dependencies/graknlabs/repositories.bzl:30]: Update to graknlabs before merging

**Comment by alexjpwalker** [comment]
Do not merge until readme issue is fixed

**Review by lolski — APPROVED** [review]

**Review by lolski — APPROVED** [review]

**Comment by alexjpwalker** [comment]
Readme issue is now fixed

**Pull request merged**
